pkg/analysis/commentstart/testdata/src/a/a.go

```diff
@@ -32,3 +46,13 @@ type CommentStartTestStruct struct {
 
 // DoNothing is used to check that the analyser doesn't report on methods.
 func (CommentStartTestStruct) DoNothing() {}
+
+type unexpectedStruct struct {
```

Comment:
Unexported?

Reply:
Sorry.
That's right. Fixed at https://github.com/JoelSpeed/kal/pull/40/commits/3302613d9af638d64595f99e808515f866fb0a93.